cli/scan-gh-bb.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const dotenv = require('dotenv')
+const { getSecurityVulnerabilitiesForMultiple } = require('../index')
+const { writeToCsv } = require('../src/csv')
+const { exitWithError } = require('../src/error')
+
+dotenv.config()
+
+const bbUsername = process.env.BITBUCKET_USERNAME
+const bbAppPassword = process.env.BITBUCKET_APP_PASSWORD
+const ghToken = process.env.GITHUB_TOKEN
+
+const ghOrg = process.argv[2]
+const bbWorkspace = process.argv[3]
+
+try {
+  assert(
+    process.argv.length > 2,
+    `Usage:\n  node cli/scan-gh-bb.js githuborganization bitbucketworkspace`
+  )
+  
+  assert(bbUsername, 'Please provide a BITBUCKET_USERNAME environment variable')
+  assert(bbAppPassword, 'Please provide a BITBUCKET_APP_PASSWORD environment variable')
+  assert(ghToken, 'Please provide a GITHUB_TOKEN environment variable to check for security vulnerabilities')
+
+  getSecurityVulnerabilitiesForMultiple({
+    bitbucket: {
+      username: bbUsername,
+      appPassword: bbAppPassword,
+      workspace: bbWorkspace,
+    },
+    github: {
+      organization: ghOrg,
+      token: ghToken,
+    },
+  }).then(writeToCsv).catch(exitWithError)
```

Comment:
would prefer to see an `async/await` approach here...looks like it would save some code too.

Reply:
I would too... but we don't have root-level async/await yet (that I know of)

Reply:
ok, not yet.